scripts/loadable_app/loadable_app.sh

```diff
@@ -0,0 +1 @@
+gcc -static -m32 -emain -nostartfiles -fpic -Wl,-N hello.c -o hello
```

Comment:
I suggest to replace `hello.c -o hello` with `"$@"` that allow to pass usual gcc options to the script. For example, next could be run from root dir, compiling hello.c example and placing result directly conf/rootfs 
```sh
$ ./scripts/loadable_app/loadable_app.sh scripts/loadable_app/hello.c -o conf/rootfs/hello
```